src/azul/plugins/repository/tdr/__init__.py

```diff
@@ -226,7 +226,7 @@ def _tdr(cls):
         return TDRClient.with_service_account_credentials()
 
     def lookup_source_id(self, spec: TDRSourceSpec) -> str:
-        return self.tdr.lookup_source_id(spec)
+        return self.tdr.lookup_source(spec).id
```

Comment:
1. Calling lookup_source here means that the sources are inspected pretty late in the CI process, the region/project id inspections would occur when we use the reindexing/canning scripts. IMO that's kinda late.

Reply:
Resolved at parking lot